UNRELEASED.md

```diff
@@ -1,8 +1,14 @@
 # Unreleased
+
 > Place your changes below this line.
 
+**Breaking:**
```

Comment:
For most use cases, the changes I've made don't require consumers to change their code.

I'm making this breaking, however, for the following reasons:
 - Not all icons are available in both large and small, whereas before they were.
 - If someone is specifying the icon string manually from the font mapping instead of using one of our auto-generated defined strings then it will break. The solution to this is to use one of the predefined values!